### PR TITLE
Adopt consistent case for section headings

### LINKED ISSUE #1040: Adopt consistent case for section headings
Of the 123 section headings, all except the following 37 currently use title case (some of these arguably do too: those where the only offending word is a trailing "method", 'enumeration" or "operation"). Shall we change these to title case?

- 1.1.3 New device registration
- 1.1.4 Other use cases and configurations
- 5.1.3 Create a new credential - PublicKeyCredential’s [[Create]](origin, options, sameOriginWithAncestors) method
- 5.1.4 Use an existing credential to make an assertion - PublicKeyCredential’s [[Get]](options) method
- 5.1.4.1 PublicKeyCredential’s [[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors) method
- 5.1.5 Store an existing credential - PublicKeyCredential’s [[Store]](credential, sameOriginWithAncestors) method
- 5.1.6 Preventing silent access to an existing credential - PublicKeyCredential’s [[preventSilentAccess]](credential, sameOriginWithAncestors) method
- 5.1.7 Availability of User-Verifying Platform Authenticator - PublicKeyCredential’s isUserVerifyingPlatformAuthenticatorAvailable() method
- 5.4.5 Authenticator Attachment enumeration (enum AuthenticatorAttachment)
- 5.4.6 Attestation Conveyance Preference enumeration (enum AttestationConveyancePreference)
- 5.6 Abort operations with AbortSignal
- 5.10.1 Client data used in WebAuthn signatures (dictionary CollectedClientData)
- 5.10.2 Credential Type enumeration (enum PublicKeyCredentialType)
- 5.10.4 Authenticator Transport enumeration (enum AuthenticatorTransport)
- 5.10.6 User Verification Requirement enumeration (enum UserVerificationRequirement)
- 6.1 Authenticator data
- 6.1.2 FIDO U2F signature format compatibility
- 6.2 Authenticator taxonomy
- 6.3 Authenticator operations
- 6.3.1 Lookup Credential Source by Credential ID algorithm
- 6.3.2 The authenticatorMakeCredential operation
- 6.3.3 The authenticatorGetAssertion operation
- 6.3.4 The authenticatorCancel operation
- 6.4.1 Attested credential data
- 6.4.1.1 Examples of credentialPublicKey Values encoded in COSE_Key format
- 7.1 Registering a new credential
- 7.2 Verifying an authentication assertion
- 8.2.1 Packed attestation statement certificate requirements
- 8.3.1 TPM attestation statement certificate requirements
- 9.2 Defining extensions
- 9.3 Extending request parameters
- 9.4 Client extension processing
- 9.5 Authenticator extension processing
- 14.1 De-anonymization prevention measures
- 14.2 Anonymous, scoped, non-correlatable public key credentials
- 14.3 Authenticator-local biometric recognition
- 14.7 Privacy between operating system accounts

### PR DESCRIPTION
**Pull request by emlun**
This fixes #1040.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1055.html" title="Last updated on Sep 5, 2018, 11:27 AM GMT (e29dbac)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/1055/035ea79...e29dbac.html" title="Last updated on Sep 5, 2018, 11:27 AM GMT (e29dbac)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -868,7 +868,7 @@ To support obtaining assertions via {{CredentialsContainer/get()|navigator.crede
 </xmp>
 
 
-### Create a new credential - PublicKeyCredential's `[[Create]](origin, options, sameOriginWithAncestors)` method ### {#createCredential}
+### Create a New Credential - PublicKeyCredential's `[[Create]](origin, options, sameOriginWithAncestors)` Method ### {#createCredential}
```
Comment (emlun) [inline index.bs]: POI: I'm not entirely sure we want to capitalize "Method" here and in similar headings, but I think so.

File: index.bs [hunk 2]
```diff
@@ -1655,7 +1655,7 @@ This method has no arguments and returns a Boolean value.
         authenticator by the client in its call to either {{CredentialsContainer/create()}} or {{CredentialsContainer/get()}}.
 </div>
 
-### Information about Public Key Credential (interface <dfn interface>AuthenticatorAttestationResponse</dfn>) ### {#iface-authenticatorattestationresponse}
+### Information About Public Key Credential (interface <dfn interface>AuthenticatorAttestationResponse</dfn>) ### {#iface-authenticatorattestationresponse}
```
Comment (emlun) [inline index.bs]: POI: "About"

File: index.bs [hunk 3]
```diff
@@ -5249,7 +5249,7 @@ concern in most [=[RPS]=]' threat models. For example, the [=man-in-the-middle a
 [=man-in-the-middle attack=] against a [=[RP]=] that only uses conventional password authentication.
 
 
-## credentialId Unsigned ## {#credentialIdSecurity}
+## Credential ID Unsigned ## {#credentialIdSecurity}
```
Comment (emlun) [inline index.bs]: POI

File: index.bs (hunk 2 above)
Comment (emlun) [reply index.bs]: POI: I did not capitalize the parenthesized "interface" here, or the "enum" or "dictionary" in similar headings.

**Comment by nadalin** [comment]
@equalsJeffH @selfissued Please review

**Comment by emlun** [comment]
@equalsJeffH please let us know before 2018-09-11T10:00 UTC if you also want to review this before we merge it.

**Pull request merged**